docs/iris/src/developers_guide/documenting/whats_new_contributions.rst

```diff
@@ -22,28 +56,49 @@ which improved Iris in some way. As such, a single Iris Pull Request may
 contain multiple changes that are worth highlighting as contributions to the
 what's new document.
 
+The appropriate contribution for a pull request might in fact be an addition or
+change to an existing "What's New" entry.
+
 Each contribution will ideally be written as a single concise bullet point
-in a reStructuredText format with a trailing blank line.  For example::
+in a reStructuredText format. Where possible do not exceed **column 80** and
+ensure that any subsequent lines of the same bullet point are aligned with the
+first. The content should target an Iris user as the audience. The required
+content, in order, is as follows:
+
+* Names of those who contributed the change. These should be their GitHub
+  display name, or if that is not available use their GitHub user name. Link
+  the name to their GitHub profile. E.g.
+  ```Bill Little <https://github.com/bjlittle>`_ and
+  `tkknight <https://github.com/tkknight>`_ changed...``
 
-  * Fixed :issue:`9999`.  Lorem ipsum dolor sit amet, consectetur adipiscing 
-    elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
-    <blank line>
+* The new/changed behaviour
 
-Note that this example also cites the related issue, optionally you may also
-include the pull request using the notation ``:pull:`9999```.  Where possible
-do not exceed **column 80** and ensure that any subsequent lines
-of the same bullet point is aligned with the first.  
+* Context to the change. Possible examples include: what this fixes, why
+  something was added, issue references (e.g. ``:issue:`9999```), more specific
+  detail on the change itself.
 
-The content of the bullet point should highlight the change that has been made
-to Iris, targeting an Iris user as the audience.
+* Pull request references, bracketed, following the final period. E.g.
+  ``(:pull:`1111`, :pull:`9999`)``
 
-For inspiration that may include adding links to code please examine past
-what's :ref:`iris_whatsnew` entries.  
+* A trailing blank line (standard reStructuredText bullet format)
+
+For example::
+
+  * `Bill Little <https://github.com/bjlittle>`_ and
+    `tkknight <https://github.com/tkknight>`_ changed changed argument ``x``
+    to be optional in :class:`~iris.module.class` and
+    :meth:`iris.module.method`. This allows greater flexibility as requested in
+    :issue:`9999`. (:pull:`1111`, :pull:`9999`)
+
+
+The above example also demonstrates some of the possible syntax for including
+links to code. For more inspiration on possible content and references please
```

Comment:
There should be a comma after "references".

Reply:
3138a01bbee3e9e2c67644ff9b4eae4eddde73aa